src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -88,8 +94,7 @@ export default class ZipPaymentStrategy implements PaymentStrategy {
             } catch (error) {
                 if (error instanceof RequestError && error.body.status === 'additional_action_required') {
                     return new Promise(() => {
-                        const { redirect_url } = error.body.additional_action_required.data;
-                        window.location.replace(redirect_url);
+                        window.location.replace(redirectUrl);
```

Comment:
Can we do similar here https://github.com/bigcommerce/checkout-sdk-js/blob/e348c5b688fc6512b4a25b17efd9680a324f4395/src/payment/strategies/quadpay/quadpay-payment-strategy.ts#L72